src/Encoders/OffloadEncoderEncryptedAes256.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Aol\Offload\Encoders;
+
+class OffloadEncoderEncryptedAes256 extends OffloadEncoderEncrypted
```

Comment:
What would oyu think about using lib-sodium-compat here or if somebody wants that they can add their own Encrypted encoder.